packages/livekit-server-sdk/src/WebhookReceiver.ts

```diff
@@ -67,7 +68,7 @@ export class WebhookReceiver {
       const claims = await this.verifier.verify(authHeader);
       // confirm sha
       const encoder = new TextEncoder();
-      const hash = await crypto.subtle.digest('SHA-256', encoder.encode(body));
+      const hash = await digest('SHA-256', encoder.encode(body));
```

Comment:
instead of converting this back and forth, we can pass the body directly and only do the textencoding part in the global webcrypto path